sentences/pl/_common.yaml

```diff
@@ -24,9 +24,211 @@ responses:
+        "group": "Żadna grupa nie została udostępniona",
+        "humidifier": "Żaden nawilżacz nie został udostępniony",
+        "image": "Żaden obraz nie został udostępniony",
+        "image_processing": "Żadne przetważanie obrazu nie zostało udostępnione",
+        "input_boolean": "Żadne pole przełącznika nie zostało udostępnione",
+        "input_button": "Żadne pole przycisku nie zostało udostępnione",
+        "input_datetime": "Żadne pole daty i godziny nie zostało udostępnione",
+        "input_number": "Żadne pole numeryczne nie zostało udostępnione",
+        "input_select": "Żadne pole wyboru nie zostało udostępnione",
+        "input_text": "Żadne pole tekstowe nie zostało udostępnione",
+        "lawn_mower": "Żadna kosiarka nie została udostępniona",
+        "light": "Żadne światło nie zostało udostępnione",
+        "lock": "Żaden zamek nie został udostępniony",
+        "media_player": "Żaden odtwarzacz multimedialny nie został udostępniony",
+        "notify": "Żadne powiadomienie nie zostało udostępnione",
+        "number": "Żadna liczba nie została udostępniona",
+        "person": "Żadna osoba nie została udostępniona",
+        "remote": "Żaden pilot nie został udostępniony",
+        "scene": "Żadna scena nie została udostępniona",
+        "schedule": "Żaden harmonogram nie został udostępniony",
+        "script": "Żaden skrypt nie został udostępniony",
+        "select": "Żadna opcja wyboru nie została udostępniona",
+        "sensor": "Żaden sensor nie został udostępniony",
+        "siren": "Żadna syrena nie została udostępniona",
+        "stt": "Żaden serwis zamiany mowy na tekst nie został udostępniony",
+        "sun": "Żadna konfiguracja słońca nie została udostępniona",
+        "switch": "Żaden przełącznik nie został udostępniony",
+        "tags": "Żaden tag nie został udostępniony",
+        "text": "Żaden tekst nie został udostępniony",
+        "time": "Żaden czas nie został udostępniony",
+        "timer": "Żaden minutnik nie został udostępniony",
+        "todo": "Żadna lista zadań nie została udostępniona",
+        "tts": "Żaden serwis zamiany tekst na mowę nie został udostępniony",
+        "update": "Żadna aktualizacja nie została udostępniona",
+        "vacuum": "Żaden odkurzacz nie został udostępniony",
+        "valve": "Żaden zawór nie został udostępniony",
+        "wake_word": "Żadne słowo budzenia nie zostało udostępnione",
+        "water_heater": "Żaden zbiornik ciepłej wody nie został udostępniony",
+        "weather": "Żaden serwis pogodowy nie został udostępnion",
+        "zone": "Żadna strefa nie została udostępniona", 
+        } %}
+      {% if domain in translations -%}
+        {{ translations[domain] }}
+      {%- else -%}
+        Domena {{ domain }} nie została udostępniona
+      {%- endif %}
+    no_domain_in_area_exposed: |
+      {% set translations = {
+        "alert": "żaden alert nie został udostępniony",
```

Comment:
In `no_domain_exposed` block all sentences started with a capital letter, here all all done with lowercase, is that itended? Probably irrelevant?

Reply:
It is correct because on the bottom you need add extra information: `W obszarze {{ area }} {{ translations[domain] }}`